test/neural_net/style_transfer.cxx

```diff
@@ -0,0 +1,138 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#define BOOST_TEST_MODULE
```

Comment:
Per the docs, this macro should define the name of the test suite.

https://www.boost.org/doc/libs/1_65_0/libs/test/doc/html/boost_test/tests_organization/test_suite/master_test_suite.html#ref_BOOST_TEST_MODULE